sdk/create-dapp/templates/react-e2e-counter/src/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const PACKAGE_ID =
+  "0x224f49ba0ca915646c3e40bab06259ce1c27516f2873a01e560b43e3e2543ed2";
```

Comment:
Need to write docs on replacing this.  I don't think we have a good story other than just telling people how to publish then find the ID in the output 😢

Reply:
I feel your pain, we deal with this on SuiFrens and maintain a good number of new/upgraded package IDs haha